website/components/DocLayout.js

```diff
@@ -49,6 +50,9 @@ function Headings({ headings }) {
 
   return (
     <Box
+      as="nav"
+      id="secondary-navigation"
+      aria-labelledby="toc-title"
```

Comment:
This is important when we have multiple `nav` in the page, so they can be differentiated from one another.